Crawler/ExtensionMethods/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace Crawler.ExtensionMethods
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static TConfig ConfigurePOCO<TConfig>(this IServiceCollection services, IConfiguration configuration) where TConfig : class, new()
+        {
+            if (services == null) throw new ArgumentNullException(nameof(services));
+            if (configuration == null) throw new ArgumentNullException(nameof(configuration));
+
+            var config = new TConfig();
+            configuration.Bind(config);
+            services.AddSingleton(config);
+
+            return config;
+        }
+    }
+}
```

Comment:
It's pretty cool, but didn't you say that the purpose of this class is to replace the need to use IOptions<T>?

Reply:
Somehow even though I keep deleting it it comes back -_-

Reply:
Fucking fixed